resources/views/welcome.blade.php

```diff
@@ -71,8 +71,7 @@
                     @auth
                         <a href="{{ url('/reading-items') }}">Home</a>
                     @else
-                        <a href="{{ route('login') }}">Login</a>
-                        <a href="{{ route('register') }}">Register</a>
```

Comment:
We need to remove the register route from the listed Routes as well. Hiding the action from view would

Reply:
Created a tracking #7. Will target separately.